components/library/fastcgi/pkg5

```diff
@@ -1,9 +1,10 @@
 {
     "dependencies": [
         "SUNWcs",
+        "shell/ksh93",
         "system/library",
-        "system/library/g++-7-runtime",
-        "system/library/gcc-7-runtime"
+        "system/library/g++-10-runtime",
+        "system/library/gcc-10-runtime"
     ],
```

Comment:
cxxtools and fastcgi both have wrong metadata (gcc-10 vs gcc-7). Shall I process nevertheless?

Reply:
I can fix them otherwise you can just revert form 10 to 7 manually if/when you push them. As you prefer :)